ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -454,28 +454,44 @@ function ListRules(props) {
         rule.product || (rule.scheduledChange && rule.scheduledChange.product);
       const ruleScheduledChangeChannel =
         rule.scheduledChange && rule.scheduledChange.channel;
-      let ruleChannel;
+      const ruleChannel = rule.channel;
 
       if (ruleProduct !== productFilter) {
         return false;
       }
 
-      if (ruleScheduledChangeChannel) {
-        if (ruleScheduledChangeChannel.includes('*')) {
-          ruleChannel = rule.channel && ruleScheduledChangeChannel;
-        } else {
-          ruleChannel = ruleScheduledChangeChannel && rule.channel;
+      if (channelFilter) {
```

Comment:
I think you could write this as `if (!channelFilter) return false;` here so that on the last line of this block you change it to `return ruleChannelMatches && scChannelMatches;` (and remove the condition on line 494). :)

Reply:
Or would it be `return ruleChannelMatches || scChannelMatches;`? Not entirely clear on that - it sounded like in the relounge channel maybe it should be OR logic??

Reply:
Definitely `||` (we want the rule to be displayed if it's _current_ channel matches OR it matches what's in a scheduled change, if it exists).